### ISSUE TITLE
QT: Unable to parse styles for QToolbar

### ISSUE THREAD

**Issue by alamshafil** [AvianNetwork/Avian#92]
When running `avian-qt` this always appears in the `debug.log`:

```
GUI: Could not parse stylesheet of object QToolBar(0x564910de90a0)
GUI: Could not parse stylesheet of object QToolBar(0x564910de90a0)
```

The stylesheet for QToolBar is defined below *(light and dark)*:

https://github.com/AvianNetwork/Avian/blob/5aa0e4aa4040c1f3f1f71b0aa1a6edf2e86875d2/src/qt/res/css/Light.css#L896-L943

**Comment by alamshafil** [comment]
This is the issue:
https://github.com/AvianNetwork/Avian/blob/5aa0e4aa4040c1f3f1f71b0aa1a6edf2e86875d2/src/qt/aviangui.cpp#L686-L696

**Comment by alamshafil** [comment]
Issue fixed in #96
